websites/S/Stoodi/metadata.json

```
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "890325980126789663",
+		"name": "Luiz ♡"
+	},
+	"service": "Stoodi",
+	"description": {
+		"en": "Stoodi is a platform to prepare you for exams and ENEM in Brazil.",
+		"pt_BR": "Stoodi é uma plataforma para te preparar para provas e ENEM no Brazil"
+	},
+	"url": "www.stoodi.com.br",
+	"regExp": "([a-z0-9]+)[.]stoodi.com.br",
```

Consistent regex, unless you meant to use the wildcard `.`
```suggestion
	"regExp": "([a-z0-9]+)[.]stoodi[.]com[.]br",
```